data.json

```diff
@@ -73,22 +73,22 @@
       "description": {{ description | jsonify }},
       {% endif %}
       "keyword":[
-      {%- for tag in dataset.tags -%}
-         {{ tag | jsonify }}{%- unless forloop.last -%},{%- endunless -%}
-      {%- endfor -%}
-      {%- if category -%},{%- endif -%}
-      {%- for cat in dataset.categories -%}
-         {%- assign cat_split = cat | split: " / " -%} 
-         {%- if cat_split.size > 1 -%}
-           {%- for cs in cat_split -%}
-              {{ cs | downcase | jsonify }}
-              {%- unless forloop.last -%},{%- endunless -%}
-           {%- endfor -%}
-         {%- else -%}
-            {{ cat | downcase | jsonify }}
-         {%- endif -%}
-         {%- unless forloop.last -%},{%- endunless -%}
-      {%- endfor -%}
+         {%- for cat in dataset.categories -%}
```

Comment:
one thing i actually can't explain is why this works with either dataset.categories or dataset.category... where are we defining categories on the dataset? is this some evil jekyll magic?

Reply:
well sure, but that doesn't explain why it works to reference the value by `dataset.categories` since the key on the yaml and the schema alike is `category`. at the same time, this is working afaict 🤔

@BryanQuigley do you have an explanation?

Reply:
I do not. I'm far from experienced in this part - as me messing up testing this PR shows :(